### PR TITLE
Revise recent edits to README

### PR DESCRIPTION
**Pull request by cbeams**
This change revises @ripcurlx's recent updates to the README in commits
d8c74ef6c07d9ea283a6babe14eebad43ff84ce9 and 58239e5728c75b56a5ff3926d2b9110b68e770a8.

Primarily it is about structure and concision. There are a number of
places where AsciiDoc syntax is updated to be more idiomatic in its
usage, a number of typos have been fixed, and certain passages have been
reworded. Several points in the document are still unclear to me as a
reader, and I'll point each of them out in dedicated comments on the
pull request I am creating for this commit.

### CODE REVIEW

**Comment by cbeams** [comment]
@ripcurlx, I missed your latest edit to the file, thus causing a merge conflict here. I'll fix that up. You can go over my changes in the meantime if you like.

**Comment by cbeams** [comment]
Actually, @ripcurlx, I see that the main thing you did was add `docs/growth-meeting.adoc`. That's fine, and I'll work in a link to that doc from the revised version of the README after I merge it.

For now, could you just go over the changes you see here? Make sure to review the rendered version to really make sure it's OK.

File: README.adoc [hunk 1]
```diff
@@ -1,155 +1,142 @@
...
+
+==== Find Levers
 
-## Finding Levers
-Goal is to find the **highest impact area** that we can **focus on right now** given limited resources.
+The goal is to find the **highest impact area** that we can **focus on right now** given limited resources.
 
-**Evaluate** +
-To evaluate each goal we should think about following key points:
+To evaluate each goal, think about following key points:
```
Comment (cbeams) [inline README.adoc]: @ripcurlx, the bullet points below don't make much sense to me. They don't have sufficient context to explain what they're about. Perhaps you could add a bit of text above or below that spells it out?

File: README.adoc [hunk 2]
```diff
@@ -1,155 +1,142 @@
...
 
-**Objective:**
+NOTE: Goals should change mid-OKR only very rarely.
 
-Define qualitative statement::
-+
-e.g. „Make Virality a Meaningful Channel“ +
-KR1: "1.5% Viral/Weekly Active User" +
-KR2: "2% Viral/Weekly Active User +
-KR3: "3% Viral/Weekly Active User
```
Comment (cbeams) [inline README.adoc]: I removed the multiple examples above for the more concise, single bullet point approach. I think the main point here is to communicate that Objectives are qualititative / subjective and Key Results are quantitative / concrete; hopefully the revised text lands that a little better.

File: README.adoc [hunk 3]
```diff
@@ -1,155 +1,142 @@
...
 
-## Finding Levers
-Goal is to find the **highest impact area** that we can **focus on right now** given limited resources.
+The goal is to find the **highest impact area** that we can **focus on right now** given limited resources.
 
-**Evaluate** +
-To evaluate each goal we should think about following key points:
+To evaluate each goal, think about following key points:
 
 1. Baseline
```
Comment (ripcurlx) [inline README.adoc]: Where are we today?

File: README.adoc [hunk 4]
```diff
@@ -1,155 +1,142 @@
...
 2. Ceiling
```
Comment (ripcurlx) [inline README.adoc]: What do we think the ceiling is? Form an educated guess, what we could reach if we would focus on just this area.

File: README.adoc [hunk 5]
```diff
@@ -1,155 +1,142 @@
...
-3. Sensitivity Over Time
```
Comment (ripcurlx) [inline README.adoc]: What is the impact of this goal over time? We always want to focus on impacts that have a long lasting effect vs. short term spikes.

File: README.adoc (hunk 1 above)
Comment (ripcurlx) [reply README.adoc]: Please see my comments below

File: README.adoc [hunk 6]
```diff
@@ -1,155 +1,142 @@
...
 
-**Create Experiment Issue** +
-See https://github.com/bisq-network/growth/blob/master/ISSUE_TEMPLATE.md[issue template] for guidance and inspiration.
+=== 3. Test
 
-## 3. Test
 What do we really need to do to test our assumption?
 
-Setting up a **Minimal Viable Test**
+==== Setting up a Minimally Viable Test
```
Comment (ripcurlx) [inline README.adoc]: The term "Minimal Viable Test" is inspired by the "Minimal Viable Product" of the lean startup methodology, so I think we should stick with the term "Minimal Viable Test"

**Comment by ripcurlx** [comment]
Regarding my last change. Yes, I only added a reference to the growth meeting document `docs/growth-meeting.md`.

All your other changes are a great improvement, so I'm happy to merge the branch as soon as the conflicts are resolved. :-)

**Comment by cbeams** [comment]
@ripcurlx, this is now merged, and per our chat in Slack, you can go ahead and fold in your comments, add back in the link to the growth call doc, and anything else you see fit. Thanks.

**Comment by ripcurlx** [comment]
Will do. Thanks!

**Pull request merged**